tests/PHPStan/Analyser/data/bug-5896.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types = 1);
+
+namespace Bug5896;
+
+use function PHPStan\Testing\assertType;
+
+class HelloWorld
+{
+	/**
+	 * @return array{default?: int}
+	 */
+	public function load(): array
+	{
+		return [
+		];
+	}
+}
+
+$helloWorld = new HelloWorld();
```

Comment:
Although PHPStan uses fully static reflection, I worry something might still actually load these classes because they're discovered by composer's classmap setting. Please put the test code (lines 19-24) into an anonymous function.